hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -144,12 +144,14 @@ def int_to_bytes(i, size):
         assert i >= 0
         result = bytearray(size)
         j = size - 1
+        arg = i
         while i and j >= 0:
             result[j] = i & 255
             i >>= 8
             j -= 1
         if i:
-            raise OverflowError('int too big to convert')
+            raise OverflowError('i=%r cannot be represented in %r bytes'
```

Comment:
I had a transient failure here, and couldn't replicate it.  Of course I haven't seen it since changing the message either, but maybe someday this will help :confused:

Reply:
Do you still have the stack trace around? Would be useful to have it in an issue for posterity.

Reply:
Alas, no - I expected to get a better one and haven't seen it since :sob